picard/ui/mainwindow.py

```diff
@@ -84,6 +83,15 @@
 )
 
 
+try:
+    from PyQt5 import QtMultimedia
+except Exception as e:
+    qt_multimedia_available = False
+    log.warning("{} - Internal player disabled".format(e.msg))
```

Comment:
No need to pre-format with `log.warning()` and we usually place the detailed reason after the main message.
Also `import` will raise [ImportError](https://docs.python.org/3/library/exceptions.html#ImportError)

```python
try:
    from PyQt5 import QtMultimedia
except ImportError as e:
    qt_multimedia_available = False
    log.warning('Internal player disabled: %s', e.msg)
else:
    qt_multimedia_available = True
```